pkg/utils/maputils.go

```diff
@@ -180,6 +180,16 @@ func AddToSet[K comparable](set map[K]struct{}, key K) {
 	set[key] = struct{}{}
 }
 
+func MapKeysToSlice[K comparable, V any](m map[K]V) []K {
```

Comment:
There's already `GetKeysOfMap()`, which does the same thing

Reply:
ohh got it!